commons/ava_testsuite/byzantine_gecko_network_tests.go

```diff
@@ -0,0 +1,115 @@
+package ava_testsuite
+
+import (
+	"github.com/kurtosis-tech/ava-e2e-tests/commons/ava_networks"
+	"github.com/kurtosis-tech/ava-e2e-tests/commons/ava_services"
+	"github.com/kurtosis-tech/kurtosis/commons/testsuite"
+	"github.com/palantir/stacktrace"
+	"github.com/sirupsen/logrus"
+	"time"
+)
+
+const (
+	BYZANTINE_CONFIG_ID = 2
+	BYZANTINE_USERNAME = "byzantine_gecko"
+	BYZANTINE_PASSWORD = "byzant1n3!"
+)
+// ================ Byzantine Test - Spamming Unrequested Chit Messages ===================================
+type StakingNetworkUnrequestedChitSpammerTest struct{
+	unrequestedChitSpammerImageName *string
+	normalImageName *string
+}
+func (test StakingNetworkUnrequestedChitSpammerTest) Run(network interface{}, context testsuite.TestContext) {
+	castedNetwork := network.(ava_networks.TestGeckoNetwork)
+	for i := 0; i < 4; i++ {
+		byzHighLevelClient, err := addServiceIdAsValidator(castedNetwork, i, BYZANTINE_USERNAME, BYZANTINE_PASSWORD, SEED_AMOUNT, STAKE_AMOUNT)
+		if err != nil {
+			context.Fatal(stacktrace.Propagate(err, "Failed to add Byzantine service as a validator."))
+		}
+		byzClient := byzHighLevelClient.GetLowLevelClient()
```

Comment:
Same comment as above - I'm not a fan of modifying the Gecko client itself just because the `network.GetClient` call just happens to be inside the helper function (when the client could just be passed in)